test/CollapsibleMixinSpec.js

```diff
@@ -67,13 +70,13 @@ describe('CollapsibleMixin', function () {
 
   describe('from collapsed to expanded', function(){
     beforeEach(function(){
-      instance = ReactTestUtils.renderIntoDocument(
+      instance = render(
         <Component>Panel content</Component>
       );
     });
 
     it('Should have collapsing class', function () {
-      instance.setProps({expanded:true});
+      instance.renderWithProps({expanded:true});
```

Comment:
I guess React is pretty aggressive about re-using component objects, and of course the mount point doesn't change... but I'll fix it anyway.